docs/testsuite_how_to.md

```diff
@@ -0,0 +1,204 @@
+How to Run the Java TestNG Tests and Expand the Test Suite Specification
+========================
+This page is meant to help familiarize a new user with using the Rest.li Test Suite Specification.
+It explains how to run the example Java TestNG tests, and how to add more tests to the Test Suite Specification. 
+
+For an overview of the test suite, refer to [Test Suite Overview](testsuite_overview.md).
+
+
+How to Run the Java TestNG Test Suite
+--------------------
+The Rest.li Test Suite Specification is language-independent, and it provides test data and guidelines for other test suites
+to follow. To validate that the spec's tests all pass using the Rest.li Java client binding, *and* to demonstrate how
+to use the Test Suite Specification, this project contains a Java TestNG test suite,
+ ocated in the client-testsuite/src/test/java folder.
+
+The Java TestNG test suite follows the Rest.li Test Suite Specification, and it has two types of tests: data driven
```

Comment:
This section should have two subsections:
- Test Categories (it has another two subsections)
- Run the tests
Please use heading format properly.